td-loader/fuzz/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "td-loader-fuzz"
```

Comment:
Should we move this into td-shim/fuzz?

Reply:
The cargo-fuzz command is to create a fuzz directory in the crate. It may be merged with the afl directory in the future.